### DISCUSSION TITLE
Autocomplete example

### DISCUSSION THREAD

**Discussion by DevulderJeanPaul** [krzyzanowskim/STTextView#38] [Questions]
Hi,
On TextEdit example, how call the autocomplete function, it's no clear.
Thanks,

**Comment by krzyzanowskim** [comment]
Tap `Esc` key

![Screenshot 2023-10-11 at 13 32 57](https://github.com/krzyzanowskim/STTextView/assets/758033/524f7dd4-f801-46d4-9ef4-1b86c309abfe)

completions list is returned from this delegate call

https://github.com/krzyzanowskim/STTextView/blob/523363516b06e92dddfc2e1d5e73aa2920758b35/TextEdit/ViewController.swift#L151

**Comment by DevulderJeanPaul** [comment]
Hi,
Thanks for the precision
Good work!
